java/serve/src/main/java/io/ray/serve/router/ReplicaSet.java

```diff
@@ -31,7 +38,13 @@ public class ReplicaSet {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ReplicaSet.class);
 
-  private final Map<ActorHandle<RayServeWrappedReplica>, Set<ObjectRef<Object>>> inFlightQueries;
+  /** Map<><ActorName/ObjectRefSet> */
```

Comment:
// The key is the name of the actor, and the value is a set of all flight queries objectrefs of the actor.